### PR TITLE
1118: [Improve] Language should be displayed in its own meaning as on Console page

### PR DESCRIPTION
**Pull request by sherlockvn**

### CODE REVIEW

**Comment by huynguyennovem** [comment]
There are redundant commits. Pls merge all them into one commit!

File: pubspec.yaml [hunk 1]
```diff
@@ -83,7 +83,7 @@ dependencies:
   image_picker: ^0.8.4+4
   ext_storage:
     git:
-      url: git://github.com/huynguyennovem/ext_storage_ups.git
```
Comment (huynguyennovem) [inline pubspec.yaml:86]: This change is not related to this issue. Pls remove from the commit!

**Comment by huynguyennovem** [comment]
Can you share the capture of language selector screen after this change?

**Comment by sherlockvn** [comment]
# Video demo the improvement
https://user-images.githubusercontent.com/43041967/160790257-613c6e34-327e-43a4-9cc6-fad276b52a66.mp4

# Enviroment
- Build version: 1.0.7
- Device: Google Pixel 4
- Android version: 11.0 x86
- QA Enviroment

**Comment by sherlockvn** [comment]
# Image demo for the improvement
image


# Enviroment
- Build version: 1.0.7
- Device: Google Pixel 4
- Android version: 11.0 x86
- QA Enviroment

**Review by huynguyennovem — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Comment by EvGen94** [comment]
> # Image demo for the improvement
> image

For example I can't understand what is that  - Soumen kieli

**Comment by sherlockvn** [comment]
I have added caption for each language as the design. Here is the video demo.

https://user-images.githubusercontent.com/43041967/161035960-c3a8263c-13d3-46d2-b7e2-2f1775388ca5.mp4

**Review by EvGen94 — APPROVED** [review]

**Review by huynguyennovem — APPROVED** [review]
